packages/graph/hash_graph/tests/test_data/src/entity_type/book.json

```diff
@@ -21,7 +21,11 @@
   "required": ["https://blockprotocol.org/@alice/types/property-type/name/"],
   "links": {
     "https://blockprotocol.org/@alice/types/link-type/written-by/v/1": {
-      "$ref": "https://blockprotocol.org/@alice/types/entity-type/person/v/1"
+      "oneOf": [
+        {
+          "$ref": "https://blockprotocol.org/@alice/types/entity-type/person/v/1"
+        }
+      ]
```

Comment:
```suggestion
      "oneOf": [
        {
          "$ref": "https://blockprotocol.org/@alice/types/entity-type/person/v/1"
        }
      ],
      "ordered": false,
```

Reply:
We love confusing JSON schema with way too many pieces of syntax all over the place 🎉 👏